src/app_engine/apprtc.py

```diff
@@ -590,6 +590,20 @@ def get(self):
     params = get_room_parameters(self.request, None, None, None)
     self.response.write(json.dumps(params))
 
+def checkIfRedirect(self):
+  REDIRECT_DOMAINS = [
```

Comment:
Would it make sense to move this into app_engine/constants.py even if it's only used here. Buried 600 lines down in source code is a pretty hidden place.